### PR TITLE
Minor Example Documentation Changes

### PR DESCRIPTION
**Pull request by timshell**
See code review by @jasmainak on #87

### CODE REVIEW

File: examples/plot_crcns_dataset_example.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 CRCNS DataSet Example
 ============================================
 
-A demonstration of Spykes's functionality using a dataset from CRCNS
+A demonstration of Spykes's functionality to reproduce a figure from Li et al's
```
Comment (jasmainak) [inline examples/plot_crcns_dataset_example.py]: actually I meant something more like here: https://github.com/mne-tools/mne-python/blob/master/examples/preprocessing/plot_eog_artifact_histogram.py#L9-L11

**Comment by jasmainak** [comment]
You should make the branch on your fork not at upstream

File: examples/plot_crcns_dataset_example.py (hunk 1 above)
Comment (timshell) [inline examples/plot_crcns_dataset_example.py]: hey @pavanramkumar did you or Hugo write the NeuroPop/CRCNS/Reaching Dataset examples?

Thanks!

**Comment by timshell** [comment]
Check it out, thanks @jasmainak !

**Comment by jasmainak** [comment]
Looks much better to me! @pavanramkumar good to merge?

**Pull request merged**